### PR TITLE
{phys}[foss/2017b] S4 v1.1.1-7fd00a2

### PR DESCRIPTION
**Pull request by vanzod**
Requires ~#8486~ (Lua)

### CODE REVIEW

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
cermis - Linux debian 9.9, Intel(R) Xeon(R) CPU           X5670  @ 2.93GHz, Python 2.7.13
See https://gist.github.com/ac9176c7f844c3a8d7d02411b18c0275 for a full test report.

**Comment by vanzod** [comment]
Re-trigger Travis after dependent PR merge

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
cermis - Linux debian 9.9, Intel(R) Xeon(R) CPU           X5670  @ 2.93GHz, Python 2.7.13
See https://gist.github.com/1edbf0adfb6287e5d7a7d01802870c8a for a full test report.

File: easybuild/easyconfigs/s/S4/S4-1.1.1-7fd00a2-foss-2017b.eb [hunk 1]
```diff
@@ -0,0 +1,51 @@
+easyblock = 'MakeCp'
+
+name = 'S4'
+commit = '7fd00a2'
+version = '1.1.1-%s' % commit
```
Comment (smoors) [inline easybuild/easyconfigs/s/S4/S4-1.1.1-7fd00a2-foss-2017b.eb]: normally, when using a specific commit, the commit date is used for the version.

File: easybuild/easyconfigs/s/S4/S4-1.1.1-7fd00a2-foss-2017b.eb [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+ S-matrix algorithm."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+
+# The original Makefile includes O3 optimization. It is then assumed safe
+# to use a higher optimization level
+toolchainopts = {'pic': True, 'usempi': True, 'opt': True}
+
+source_urls = ['https://github.com/victorliu/S4/archive']
+sources = [{'filename': 'S4-%%(version)s-%s.tar.gz' % commit, 'download_filename': '%s.tar.gz' % commit}]
```
Comment (smoors) [inline easybuild/easyconfigs/s/S4/S4-1.1.1-7fd00a2-foss-2017b.eb]: you have the commit twice in the filename.

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 3 out of 3 (1 easyconfigs in this PR)
nic425 - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/2988ae3a37bab833eb5f6d15821f372d for a full test report.

**Comment by smoors** [comment]
Going in, thanks @vanzod!

**Pull request merged**